client/senddecorator/configurator.go

```diff
@@ -2,19 +2,14 @@ package senddecorator
 
 import (
 	"github.com/Azure/go-autorest/autorest"
-
-	"github.com/giantswarm/azure-operator/v5/pkg/backpressure"
 )
 
-// ConfigureClient accepts backpressure instance and configures given autorest
-// Client instance with all local `autorest.SendDecorator` implementations in this
-// package.
+// WrapClient accepts variable number of SendDecorators that wrap the given
+// autorest Client.
 //
 // Existing SendDecorators are preserved, but moved to end of slice.
-func ConfigureClient(g *backpressure.Backpressure, c *autorest.Client) {
-	c.SendDecorators = append([]autorest.SendDecorator{
-		// NOTE: Order matters here since these decorators are executed in
-		// order. See: https://godoc.org/github.com/Azure/go-autorest/autorest#Client
-		RateLimitCircuitBreaker(g),
-	}, c.SendDecorators...)
+func WrapClient(c *autorest.Client, decorators ...autorest.SendDecorator) {
```

Comment:
I'm missing something. When is this function used?

Reply:
Here: https://github.com/giantswarm/azure-operator/pull/1101/files#diff-ed5adad93d3328a35e51172ed8aed66598cfef9e2c769de9c92287bf99bc2c65R192-R200